core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -582,24 +599,12 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
    */
   def updateSubmissionStatus(
     dataAccess: DataAccess
-  )(implicit executionContext: ExecutionContext): ReadWriteAction[Boolean] = {
-    val workflowRecsAction = if (costCapThreshold.isDefined) {
-      dataAccess.workflowQuery.listWorkflowRecsForSubmission(submissionId)
-    } else {
-      dataAccess.workflowQuery.listWorkflowRecsForSubmissionAndStatuses(
-        submissionId,
-        (WorkflowStatuses.queuedStatuses ++ WorkflowStatuses.runningStatuses): _*
-      )
-    }
-
-    workflowRecsAction.flatMap { workflowRecs =>
-      val nonTerminalWorkflows =
-        if (costCapThreshold.isDefined)
-          workflowRecs
-            .filterNot(wf => WorkflowStatuses.terminalStatuses.contains(WorkflowStatuses.withName(wf.status)))
-        else workflowRecs
-
-      if (nonTerminalWorkflows.isEmpty) {
+  )(implicit executionContext: ExecutionContext): ReadWriteAction[Boolean] =
```

Comment:
I've reverted `updateSubmissionStatus` to its implementation prior to the submission cost cap. This method checks on the workflows in a submission and updates the submission status to a terminal status if appropriate. The cost capped implementation would also update the submission status to `Aborting` if the submission's workflows had exceeded the cost cap, but that is no longer necessary as each workflow will be aborted independently.

Reply:
Nice that we won't have to continue asking Cromwell about the costs of finished workflows while the submission is running!